coordinator/coordinator.go

```diff
@@ -59,6 +59,20 @@ func (c *Coordinator) Run() error {
 	// incrementing or decrementing users again.
 	restTime := time.Duration(c.config.RestTimeSec) * time.Second
 
+	// TODO: considering making the following values configurable.
+
+	// The threshold at which we consider the load-test done and we are ready to
+	// give an answer. The value represent the slope of the best fine line for
+	// the gathered samples. This value approaching zero means we have found
+	// an equilibrium point.
+	stopThreshold := 0.1
```

Comment:
This is really still in *trial and error* land. I think that problem is mainly solved by using a wide enough time-range so that you don't take into account small variations which are expected. 
I think the only problem might rise if we the values for `NumUsersInc` and `NumUsersDec` start to really differ but I would advise against that.
I'd probably wait on this one. There's always time to make it more complex.

Reply:
Yeah, sounds good.